operations/services.md

```diff
@@ -95,6 +95,152 @@ Returns all services offered by the enterprise.
+            "ChannelManager": null,
+            "State": "Confirmed",
+            "Origin": "Connector",
+            "CreatedUtc": "2020-11-05T12:50:40Z",
+            "UpdatedUtc": "2020-11-06T07:59:19Z",
+            "CancelledUtc": null,
+            "StartUtc": "2020-11-05T00:00:00Z",
+            "EndUtc": "2020-11-06T00:00:00Z",
+            "ReleasedUtc": null,
+            "RequestedCategoryId": "1268c440-21c5-415d-bf58-ac87008b2bda",
+            "AssignedSpaceId": "f97a6b96-b17f-421f-9b97-ac87008b3324",
+            "AssignedResourceId": "f97a6b96-b17f-421f-9b97-ac87008b3324",
+            "AssignedSpaceLocked": false,
+            "AssignedResourceLocked": false,
+            "BusinessSegmentId": null,
+            "CompanyId": null,
+            "TravelAgencyId": null,
+            "AvailabilityBlockId": "aaaa654a4a94-4f96-9efc-86da-bd26d8db",
+            "RateId": "ed4b660b-19d0-434b-9360-a4de2ea42eda",
+            "CreditCardId": null,
+            "AdultCount": 2,
+            "ChildCount": 0,
+            "CustomerId": "c2730cbc-53ca-440d-8b30-ac87008b30af",
+            "BookerId": null,
+            "CompanionIds": [],
+            "CancellationReason": null,
+            "VoucherId": null
+        }
+    ],
+    "Availabilities": [
+        {
+            "AvailabilityBlockId": "aaaa654a4a94-4f96-9efc-86da-bd26d8db",
+            "ResourceCategoryId": "1268c440-21c5-415d-bf58-ac87008b2bda",
+            "StartUtc": "2020-11-05T23:00:00Z",
+            "EndUtc": "2020-11-06T23:00:00Z",
+            "UnitCount": 6
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `AvailabilityBlocks` | array of [Availability block](#availability-block) | optional | Availability blocks. |
+| `ServiceOrders` | array of [Reservation](reservations.md#reservation) | optional | Service orders (for example [Reservation](reservations.md#reservation)s) linked to availability blocks. |
+| `Availabilities` | array of [Availability update](#availability-update) | optional | Availability updates of availability blocks. |
+
+#### Availability block
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the availability block. |
+| `ServiceId` | string | required | Unique identifier of the [Service](#service) the block is assigned to. |
+| `RateId` | string | required | Unique identifier of the [Rate](#rate) the block is assigned to. |
+| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `ReleasedUtc` | string | required | The moment when the block and its availability is released in UTC timezone in ISO 8601 format. |
+| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the block from external system. |
+
+#### Availability update
```

Comment:
This doesn't seem like update. Isn't it more like Availability block availability? If we also use it in update then I'd rename it here and inside update, I'd use `Availability block availability`. Having update here feels weird, other way around it would be better.